netket/operator/_discrete_operator_jax.py

```diff
@@ -208,6 +209,27 @@ def to_sparse(self) -> JAXSparse:
         ij = np.concatenate((i[:, None], j[:, None]), axis=1)
         return BCOO((a, ij), shape=(n, n))
 
+    def _to_sparse_scipy(self) -> csr_matrix:
```

Comment:
is this part of an API or it's just here?

Reply:
no, I just haven't removed it yet